proto/gen/rill/runtime/v1/runtime.swagger.yaml

```diff
@@ -413,7 +413,7 @@ paths:
             title: Request message for RuntimeService.MetricsViewTimeSeries
       tags:
         - RuntimeService
-  /v1/instances/{instanceId}/metrics-views/{metricsViewName}/toplist/{dimensionName}:
+  /v1/instances/{instanceId}/metrics-views/{metricsViewName}/toplist:
```

Comment:
can a tableName or metricsViewName contain a forward slash ?
If it is possible then that would also cause issues with the path.

Reply:
Also add a check that instanceId cannot have forward slash..

Reply:
1. I asked for clarification about table name and metrics view name in the Notion doc
2. Instance IDs should not have forward slashes (when we add request param validation with https://github.com/bufbuild/protoc-gen-validate, which is in the backlog, this will be enforced)